_ontimize_web_ngx_map/configuration/04-o-geojson-layer.component-api.md

```diff
@@ -0,0 +1,7 @@
+---
+permalink: /map/configuration/o-geojson-layer/api
```

Comment:
I think this file isnt necessary and this page shout use other layout (without API tab)